chart/values.yaml

```diff
@@ -120,24 +129,24 @@ ingress:
     annotations: {}
 
     # The path for the flower Ingress
-    path: ""
+    path: "/"
```

Comment:
Both path and pathType are mandatory in the stable api. I don't think setting it by default is likely break many people's config but thought I'd point it out.